### DISCUSSION TITLE
where is the version # ?

### DISCUSSION THREAD

**Discussion by JGHBone** [visnkmr/backgroundappslist#832] [Q&A]
Sorry if this is a dumb question, but where in the interface can I find the version # of the currently installed app?  App is sideloaded on my firesticks.  Looked everywhere but can't find it.

**Comment by visnkmr** [comment]
1. Go to Firetv Home Screen

2. Find the app you would like to view the app version of, and click the menu button (three lines) on your FireTv remote

3. Side pop up will show then click “More Info”

4. In the right half of screen you can see version number.

**Reply by JGHBone** [reply]
Strangely, that didn't work for me (didn't show version number).  However, "Manage Installed Applications" under Settings, Applications DID display version number.  So all good.  Thanks.
